### ISSUE TITLE
在训练时报了CUDNN_STATUS_EXECUTION_FAILED错误

### ISSUE THREAD

**Issue by chaogehah** [PaddlePaddle/Paddle#22874]
我用的是ubuntu16.04
显卡是RTX2080Ti
我是在anaconda的虚拟环境下跑的训练，网络为ResNet50_vd
paddle1.7.0.post97 + cuda9.0 + cudnn7.3.1

2020-03-05 14:27:22,808-INFO: ------------- Configuration Arguments -------------
2020-03-05 14:27:22,808-INFO: batch_size : 16
2020-03-05 14:27:22,808-INFO: checkpoint : None
2020-03-05 14:27:22,808-INFO: class_dim : 2
2020-03-05 14:27:22,808-INFO: data_dir : ./data/ILSVRC2012/
2020-03-05 14:27:22,808-INFO: data_format : NCHW
2020-03-05 14:27:22,808-INFO: decay_epochs : 2.4
2020-03-05 14:27:22,808-INFO: decay_rate : 0.97
2020-03-05 14:27:22,808-INFO: drop_connect_rate : 0.2
2020-03-05 14:27:22,808-INFO: ema_decay : 0.9999
2020-03-05 14:27:22,808-INFO: enable_ce : False
2020-03-05 14:27:22,808-INFO: finetune_exclude_pretrained_params : None
2020-03-05 14:27:22,808-INFO: fuse_bn_act_ops : False
2020-03-05 14:27:22,808-INFO: fuse_elewise_add_act_ops : False
2020-03-05 14:27:22,808-INFO: image_mean : [0.485, 0.456, 0.406]
2020-03-05 14:27:22,808-INFO: image_shape : [3, 224, 224]
2020-03-05 14:27:22,808-INFO: image_std : [0.229, 0.224, 0.225]
2020-03-05 14:27:22,808-INFO: interpolation : None
2020-03-05 14:27:22,808-INFO: is_profiler : False
2020-03-05 14:27:22,808-INFO: l2_decay : 7e-05
2020-03-05 14:27:22,808-INFO: label_smoothing_epsilon : 0.1
2020-03-05 14:27:22,808-INFO: lower_ratio : 0.75
2020-03-05 14:27:22,808-INFO: lower_scale : 0.08
2020-03-05 14:27:22,808-INFO: lr : 0.1
2020-03-05 14:27:22,808-INFO: lr_strategy : cosine_decay
2020-03-05 14:27:22,808-INFO: max_iter : 0
2020-03-05 14:27:22,808-INFO: mixup_alpha : 0.2
2020-03-05 14:27:22,808-INFO: model : ResNet50_vd
2020-03-05 14:27:22,808-INFO: model_save_dir : output/
2020-03-05 14:27:22,808-INFO: momentum_rate : 0.9
2020-03-05 14:27:22,808-INFO: num_epochs : 200
2020-03-05 14:27:22,808-INFO: padding_type : SAME
2020-03-05 14:27:22,808-INFO: pretrained_model : None
2020-03-05 14:27:22,808-INFO: print_step : 10
2020-03-05 14:27:22,808-INFO: profiler_path : ./profilier_files
2020-03-05 14:27:22,808-INFO: random_seed : None
2020-03-05 14:27:22,808-INFO: reader_buf_size : 2048
2020-03-05 14:27:22,808-INFO: reader_thread : 8
2020-03-05 14:27:22,808-INFO: resize_short_size : 256
2020-03-05 14:27:22,808-INFO: same_feed : 0
2020-03-05 14:27:22,808-INFO: save_step : 1
2020-03-05 14:27:22,808-INFO: scale_loss : 1.0
2020-03-05 14:27:22,808-INFO: step_epochs : [30, 60, 90]
2020-03-05 14:27:22,808-INFO: test_batch_size : 8
2020-03-05 14:27:22,809-INFO: total_images : 476
2020-03-05 14:27:22,809-INFO: upper_ratio : 1.3333333333333333
2020-03-05 14:27:22,809-INFO: use_aa : False
2020-03-05 14:27:22,809-INFO: use_dali : False
2020-03-05 14:27:22,809-INFO: use_dynamic_loss_scaling : True
2020-03-05 14:27:22,809-INFO: use_ema : False
2020-03-05 14:27:22,809-INFO: use_fp16 : False
2020-03-05 14:27:22,809-INFO: use_gpu : True
2020-03-05 14:27:22,809-INFO: use_label_smoothing : 1
2020-03-05 14:27:22,809-INFO: use_mixup : 1
2020-03-05 14:27:22,809-INFO: use_se : True
2020-03-05 14:27:22,809-INFO: validate : 1
2020-03-05 14:27:22,809-INFO: warm_up_epochs : 5.0
2020-03-05 14:27:22,809-INFO: ----------------------------------------------------
W0305 14:27:26.086107 23366 device_context.cc:237] Please NOTE: device: 0, CUDA Capability: 75, Driver API Version: 10.1, Runtime API Version: 9.0
W0305 14:27:26.087620 23366 device_context.cc:245] device: 0, cuDNN Version: 7.3.
I0305 14:27:26.850368 23366 parallel_executor.cc:440] The Program will be executed on CUDA using ParallelExecutor, 1 cards are used, so 1 programs are executed in parallel.
I0305 14:27:26.869619 23366 build_strategy.cc:365] SeqOnlyAllReduceOps:0, num_trainers:1
I0305 14:27:26.908351 23366 parallel_executor.cc:307] Inplace strategy is enabled, when build_strategy.enable_inplace = True
I0305 14:27:26.920976 23366 parallel_executor.cc:375] Garbage collection strategy is enabled, when FLAGS_eager_delete_tensor_gb = 0
2020-03-05 14:27:27,033-INFO: [Pass 0, train batch 0] loss 0.71246, lr 0.10000, elapse 0.2654 sec
/home/junc-lin/anaconda3/envs/paddle_test/lib/python3.7/site-packages/paddle/fluid/executor.py:782: UserWarning: The following exception is not an EOF exception.
"The following exception is not an EOF exception.")
Traceback (most recent call last):
File "train.py", line 304, in
main()
File "train.py", line 300, in main
train(args)
File "train.py", line 250, in train
fetch_list=train_fetch_list)
File "/home/junc-lin/anaconda3/envs/paddle_test/lib/python3.7/site-packages/paddle/fluid/executor.py", line 783, in run
six.reraise(*sys.exc_info())
File "/home/junc-lin/anaconda3/envs/paddle_test/lib/python3.7/site-packages/six.py", line 703, in reraise
raise value
File "/home/junc-lin/anaconda3/envs/paddle_test/lib/python3.7/site-packages/paddle/fluid/executor.py", line 778, in run
use_program_cache=use_program_cache)
File "/home/junc-lin/anaconda3/envs/paddle_test/lib/python3.7/site-packages/paddle/fluid/executor.py", line 843, in _run_impl
return_numpy=return_numpy)
File "/home/junc-lin/anaconda3/envs/paddle_test/lib/python3.7/site-packages/paddle/fluid/executor.py", line 677, in _run_parallel
tensors = exe.run(fetch_var_names)._move_to_list()
paddle.fluid.core_avx.EnforceNotMet:

C++ Call Stacks (More useful to developers):
0 std::string paddle::platform::GetTraceBackString<char const*>(char const*&&, char const*, int)
1 paddle::platform::EnforceNotMet::EnforceNotMet(std::__exception_ptr::exception_ptr, char const*, int)
2 paddle::operators::CUDNNConvOpKernel::Compute(paddle::framework::ExecutionContext const&) const
3 std::_Function_handler<void (paddle::framework::ExecutionContext const&), paddle::framework::OpKernelRegistrarFunctor<paddle::platform::CUDAPlace, false, 0ul, paddle::operators::CUDNNConvOpKernel, paddle::operators::CUDNNConvOpKernel, paddle::operators::CUDNNConvOpKernelpaddle::platform::float16 >::operator()(char const*, char const*, int) const::{lambda(paddle::framework::ExecutionContext const&)#1}>::_M_invoke(std::_Any_data const&, paddle::framework::ExecutionContext const&)
4 paddle::framework::OperatorWithKernel::RunImpl(paddle::framework::Scope const&, paddle::platform::Place const&, paddle::framework::RuntimeContext*) const
5 paddle::framework::OperatorWithKernel::RunImpl(paddle::framework::Scope const&, paddle::platform::Place const&) const
6 paddle::framework::OperatorBase::Run(paddle::framework::Scope const&, paddle::platform::Place const&)
7 paddle::framework::details::ComputationOpHandle::RunImpl()
8 paddle::framework::details::FastThreadedSSAGraphExecutor::RunOpSync(paddle::framework::details::OpHandleBase*)
9 paddle::framework::details::FastThreadedSSAGraphExecutor::RunTracedOps(std::vector<paddle::framework::details::OpHandleBase*, std::allocatorpaddle::framework::details::OpHandleBase* > const&)
10 paddle::framework::details::FastThreadedSSAGraphExecutor::Run(std::vector<std::string, std::allocatorstd::string > const&)
11 paddle::framework::details::ScopeBufferedMonitor::Apply(std::function<void ()> const&, bool)
12 paddle::framework::details::ScopeBufferedSSAGraphExecutor::Run(std::vector<std::string, std::allocatorstd::string > const&)
13 paddle::framework::ParallelExecutor::Run(std::vector<std::string, std::allocatorstd::string > const&)

Python Call Stacks (More useful to users):
File "/home/junc-lin/anaconda3/envs/paddle_test/lib/python3.7/site-packages/paddle/fluid/framework.py", line 2525, in append_op
attrs=kwargs.get("attrs", None))
File "/home/junc-lin/anaconda3/envs/paddle_test/lib/python3.7/site-packages/paddle/fluid/layer_helper.py", line 43, in append_op
return self.main_program.current_block().append_op(*args, **kwargs)
File "/home/junc-lin/anaconda3/envs/paddle_test/lib/python3.7/site-packages/paddle/fluid/layers/nn.py", line 1405, in conv2d
"data_format": data_format,
File "/home/junc-lin/Downloads/models-develop/PaddleCV/image_classification/models/resnet_vd.py", line 146, in conv_bn_layer
bias_attr=False)
File "/home/junc-lin/Downloads/models-develop/PaddleCV/image_classification/models/resnet_vd.py", line 67, in net
name='conv1_1')
File "/home/junc-lin/Downloads/models-develop/PaddleCV/image_classification/build_model.py", line 98, in _mixup_model
net_out = model.net(input=image, class_dim=args.class_dim)
File "/home/junc-lin/Downloads/models-develop/PaddleCV/image_classification/build_model.py", line 125, in create_model
loss_out = _mixup_model(data, model, args, is_train)
File "train.py", line 65, in build_program
data_loader, loss_out = create_model(model, args, is_train)
File "train.py", line 166, in train
args=args)
File "train.py", line 300, in main
train(args)
File "train.py", line 304, in
main()

Error Message Summary:
Error: An error occurred here. There is no accurate error hint for this error yet. We are continuously in the process of increasing hint for this kind of error check. It would be helpful if you could inform us of how this conversion went by opening a github issue. And we will resolve it with high priority.

New issue link: https://github.com/PaddlePaddle/Paddle/issues/new
Recommended issue content: all error stack information
[Hint: CUDNN_STATUS_EXECUTION_FAILED] at (/paddle/paddle/fluid/operators/conv_cudnn_op.cu:286)
[operator < conv2d > error]

**Comment by yaoxuefeng6** [comment]
https://github.com/PaddlePaddle/Paddle/issues/15140 看看这个有帮助吗

**Comment by chaogehah** [comment]
> #15140 看看这个有帮助吗

他这个是因为cuda跟cudnn的问题，可是我的cuda跟cudnn是安装好的，之前就用的这个版本跑过tensorflow框架的网络，都是能跑的，但是这个就报错了

**Comment by chaogehah** [comment]
> #15140 看看这个有帮助

我刚刚试了一下用cuda10.0的版本，但是执行conda install paddle-gpu之后，他安装的是cudnn7.3.1，我再次执行文件这次报错改成了[Hint: CUDNN_STATUS_BAD_PARAM] at (/paddle/paddle/fluid/operators/batch_norm_op.cu:287)
[operator < batch_norm > error]

#15140 里用的是cudnn7.4，但是我在conda里安装了paddle之后就没法安装cudnn7.4了，会一直报错，所以请问有什么其他的解决办法吗？

**Comment by yaoxuefeng6** [comment]
可以尝试升级cudnn后 再重新安装paddle试试哈

**Comment by chaogehah** [comment]
> 可以尝试升级cudnn后 再重新安装paddle试试哈

我安装时报了这个错误，请问是anaconda的问题吗？百度了一下好像是版本问题，没什么特别好的解决办法

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - cudnn=7.4.0

Current channels:

  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

**Comment by JepsonWong** [comment]
in https://github.com/pytorch/pytorch/issues/17543 ：
CUDA 9 and RTX 2080 Ti simply aren't compatible and dont play well togethere.
An older CuDNN version working is likely a side-effect rather than expectation.
Use CUDA10 and CUDA10 versions of CuDNN etc. for RTX 2080 which is Turing architecture.

Can you try upgrading CUDA to 10.0?

Can you use docker? Can you try the following docker environment？
- docker pull paddlepaddle/paddle:latest-gpu-cuda10.0-cudnn7-dev

**Comment by chaogehah** [comment]
> in [pytorch/pytorch#17543](https://github.com/pytorch/pytorch/issues/17543) ：
> CUDA 9 and RTX 2080 Ti simply aren't compatible and dont play well togethere.
> An older CuDNN version working is likely a side-effect rather than expectation.
> Use CUDA10 and CUDA10 versions of CuDNN etc. for RTX 2080 which is Turing architecture.
> 
> Can you try upgrading CUDA to 10.0?
> 
> Can you use docker? Can you try the following docker environment？
> 
> * docker pull paddlepaddle/paddle:latest-gpu-cuda10.0-cudnn7-dev

1.我尝试了升级到CUDA10.0，并且安装了paddle的cuda10.0的版本，但是当我输入完conda install paddlepaddle-gpu cudatoolkit=10.0并且安装完成后，其安装的cudnn版本为7.3.1无法使用，并且我在当前环境下无法安装别的版本的cudnn，一安装就出错。然后我尝试先安装cudnn7.6.0，但是装完之后paddle就无法安装了，这是我目前很头疼的一个问题，就是paddle跟cudnn的无法同时安装正确的版本，请问有别的什么好的办法吗？

2.docker的话我自己没有用过，然后我对conda用的稍微熟练一点，如果能在conda下解决这个问题的话是最好的，若是实在没办法我再想想能不能用docker安装一下

**Comment by JepsonWong** [comment]
> > in [pytorch/pytorch#17543](https://github.com/pytorch/pytorch/issues/17543) ：
> > CUDA 9 and RTX 2080 Ti simply aren't compatible and dont play well togethere.
> > An older CuDNN version working is likely a side-effect rather than expectation.
> > Use CUDA10 and CUDA10 versions of CuDNN etc. for RTX 2080 which is Turing architecture.
> > Can you try upgrading CUDA to 10.0?
> > Can you use docker? Can you try the following docker environment？
> > 
> > * docker pull paddlepaddle/paddle:latest-gpu-cuda10.0-cudnn7-dev
> 
> 1.我尝试了升级到CUDA10.0，并且安装了paddle的cuda10.0的版本，但是当我输入完conda install paddlepaddle-gpu cudatoolkit=10.0并且安装完成后，其安装的cudnn版本为7.3.1无法使用，并且我在当前环境下无法安装别的版本的cudnn，一安装就出错。然后我尝试先安装cudnn7.6.0，但是装完之后paddle就无法安装了，这是我目前很头疼的一个问题，就是paddle跟cudnn的无法同时安装正确的版本，请问有别的什么好的办法吗？
> 
> 2.docker的话我自己没有用过，然后我对conda用的稍微熟练一点，如果能在conda下解决这个问题的话是最好的，若是实在没办法我再想想能不能用docker安装一下

安装paddle cuda 10.0版本，安装的cudnn版本为7.3.1无法使用的表现是什么？
先安装cudnn7.6.0，为什么paddle无法安装？有什么报错么？

**Comment by chaogehah** [comment]
> > > in [pytorch/pytorch#17543](https://github.com/pytorch/pytorch/issues/17543) ：
> > > CUDA 9 and RTX 2080 Ti simply aren't compatible and dont play well togethere.
> > > An older CuDNN version working is likely a side-effect rather than expectation.
> > > Use CUDA10 and CUDA10 versions of CuDNN etc. for RTX 2080 which is Turing architecture.
> > > Can you try upgrading CUDA to 10.0?
> > > Can you use docker? Can you try the following docker environment？
> > > 
> > > * docker pull paddlepaddle/paddle:latest-gpu-cuda10.0-cudnn7-dev
> > 
> > 
> > 1.我尝试了升级到CUDA10.0，并且安装了paddle的cuda10.0的版本，但是当我输入完conda install paddlepaddle-gpu cudatoolkit=10.0并且安装完成后，其安装的cudnn版本为7.3.1无法使用，并且我在当前环境下无法安装别的版本的cudnn，一安装就出错。然后我尝试先安装cudnn7.6.0，但是装完之后paddle就无法安装了，这是我目前很头疼的一个问题，就是paddle跟cudnn的无法同时安装正确的版本，请问有别的什么好的办法吗？
> > 2.docker的话我自己没有用过，然后我对conda用的稍微熟练一点，如果能在conda下解决这个问题的话是最好的，若是实在没办法我再想想能不能用docker安装一下
> 
> 安装paddle cuda 10.0版本，安装的cudnn版本为7.3.1无法使用的表现是什么？
> 先安装cudnn7.6.0，为什么paddle无法安装？有什么报错么？

先安装paddle时，报错是这样的

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: - 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
Examining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip openssl py-cpuinfo paddlepaddle-gpu pillow mkl_raExamining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip openssl py-cpuinfo paddlepaddle-gpu pillow mkl_raExamining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip py-cpuinfo sqlite paddlepaddle-gpu pillow mkl_ranExamining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip py-cpuinfo sqlite paddlepaddle-gpu pillow mkl_ranExamining conflict for pycparser pyyaml olefile gstreamer python kiwisolver graphviz numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cairo cycler protobuf pango chardet requests certifi matplotlib-base sip py-cpuinfo paddlepaddle-gpu pillow mkl_random pip dbus gst-plugins-base qt urllib3 idna nltk wheel glib libffi six cryptography matplotlib numpy-base cffi harfExamining conflict for pycparser pyyaml olefile gstreamer python kiwisolver graphviz numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cairo cycler protobuf pango chardet requests certifi matplotlib-base sip py-cpuinfo paddlepaddle-gpu pillow mkl_random pip dbus gst-plugins-base qt urllib3 idna nltk wheel glib libffi six cryptography matplotlib numpy-base cffi harfExamining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip py-cpuinfo paddlepaddle-gpu pillow mkl_random pip urllib3 idna nltk wheel six cryptography matplotlib numpy-base cffi pyqt pyparsing:   5%| | 4/87 [00:12<02:50,  2.06s/it]                   Examining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip py-cpuinfo paddlepaddle-gpu pillow mkl_random pipExamining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip py-cpuinfo paddlepaddle-gpu pillow mkl_random pipExamining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip py-cpuinfo paddlepaddle-gpu pillow mkl_random pipExamining conflict for pycparser pyyaml olefile gstreamer jpeg python kiwisolver libxcb graphviz libgcc-ng decorator libxml2 python-dateutil icu zlib mkl-service numpy tornado mkl_fft scipy pixman libpng pysocks pyopenssl asn1crypto libedit zstd setuptools cairo cycler fontconfig fribidi protobuf pango chardet freetype certifi requests matplotlib-base sip yaml expat openssl py-cpuinfo libtiff readline graphite2 sqlite paddlepaddle-gpu pillow xz mkl_random pip dbus gst-plugins-base qt urllib3 tk idna nltk wheel glib libuuid libffi six cryptography libprotobufExamining conflict for pycparser pyyaml olefile gstreamer jpeg python kiwisolver libxcb graphviz libgcc-ng decorator libxml2 python-dateutil icu zlib mkl-service numpy tornado mkl_fft scipy pixman libpng pysocks pyopenssl asn1crypto libedit zstd setuptools cairo cycler fontconfig fribidi protobuf pango chardet freetype certifi requests matplotlib-base sip yaml expat openssl py-cpuinfo libtiff readline graphite2 sqlite paddlepaddle-gpu pillow xz mkl_random pip dbus gst-plugins-base qt urllib3 tk idna nltk wheel glib libuuid libffi six cryptography libprotobufExamining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip py-cpuinfo readline paddlepaddle-gpu pillow mkl_random pip urllib3 idna nltk wheel six cryptography matplotlib numpy-base cffi pyqt pyparsing:   8%| | 7/87 [00:23<04:18,  3.23s/it]                                                                                                                                                       Examining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip py-cpuinfo readline paddlepaddle-gpu pillow mkl_rExamining conflict for pycparser pyyaml olefile gstreamer python kiwisolver graphviz libxml2 decorator python-dateutil numpy zlib mkl-service tornado mkl_fft scipy pysocks libpng pyopenssl asn1crypto zstd setuptools cairo cycler fontconfig protobuf pango chardet freetype certifi requests matplotlib-base sip py-cpuinfo libtiff paddlepaddle-gpu pillow mkl_random pip dbus gst-plugins-base qt urllib3 tk idna nltk wheel glibExamining conflict for pycparser pyyaml olefile gstreamer python kiwisolver graphviz libxml2 decorator python-dateutil numpy zlib mkl-service tornado mkl_fft scipy pysocks libpng pyopenssl asn1crypto zstd setuptools cairo cycler fontconfig protobuf pango chardet freetype certifi requests matplotlib-base sip py-cpuinfo libtiff paddlepaddle-gpu pillow mkl_random pip dbus gst-plugins-base qt urllib3 tk idna nltk wheel glibExamining conflict for pycparser pyyaml olefile gstreamer python kiwisolver graphviz libxml2 decorator python-dateutil numpy mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto zstd setuptools cairo cycler fontconfig protobuf pango chardet requests certifi matplotlib-base sip py-cpuinfo libtiff paddlepaddle-gpu pillow xz mkl_random pip gst-plugins-base qt urllib3 idna nltk wheel six cryptography matplotlib nuExamining conflict for pycparser pyyaml olefile gstreamer python kiwisolver graphviz libxml2 decorator python-dateutil numpy mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto zstd setuptools cairo cycler fontconfig protobuf pango chardet requests certifi matplotlib-base sip py-cpuinfo libtiff paddlepaddle-gpu pillow xz mkl_random pip gst-plugins-base qt urllib3 idna nltk wheel six cryptography matplotlib nuExamining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto ca-certificates setuptools cycler protobuf chardet requests sip certifi openssl py-cpuinfo paddlepaddle-gpu pillow mkl_random pip qt urllib3 idna nltk wheel six cryptography matplotlib numpy-base cffi pyqt pyparsing:  11%| | 10/87 [00:35<04:40,  3.64s/it]       Examining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto ca-certificates setuptools cycler protobuf chardet requests sip certifi openssl py-cpuinfo paddlepaddle-gpu pillow mkl_raExamining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto libedit setuptools cycler protobuf chardet requests matplotlib-base certifi sip py-cpuinfo readline sqlite paddlepaddle-gpu pillow mkl_random pip urllib3 idna nltk wheel six cryptography ncurses matplotlib numpy-base cffi pyqt pyparsing:  13%|▏| 11/87 [00:38<04:Examining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto libedit setuptools cycler protobuf chardet requests matplotlib-base certifi sip py-cpuinfo readline sqlite paddlepaddle-gpu pillow mkl_random pip urllib3 idna nltk wheel six cryptography ncurses matplotlib numpy-base cffi pyqt pyparsing:  14%|▏| 12/87 [00:38<04:Examining conflict for pycparser pyyaml olefile gstreamer python kiwisolver graphviz libxml2 decorator python-dateutil icu numpy mkl-service tornado mkl_fft libstdcxx-ng scipy pysocks pyopenssl asn1crypto libedit zstd setuptools cairo cycler fontconfig protobuf pango chardet requests certifi matplotlib-base sip expat py-cpuinfo libtiff readline graphite2 paddlepaddle-gpu pillow mkl_random pip dbus gst-plugins-base qt urllib3 idna nltk wheel glib libffi six cryptography libprotobuf ncurses pcre matplotlib numpy-base cffi harfbuzz pyqt pyparsing:  14%|▏| 12/87Examining conflict for pycparser pyyaml olefile gstreamer python kiwisolver graphviz libxml2 decorator python-dateutil icu numpy mkl-service tornado mkl_fft libstdcxx-ng scipy pysocks pyopenssl asn1crypto libedit zstd setuptools cairo cycler fontconfig protobuf pango chardet requests certifi matplotlib-base sip expat py-cpuinfo libtiff readline graphite2 paddlepaddle-gpu pillow mkl_random pip dbus gst-plugins-base qt urllib3 idna nltk wheel glib libffi six cryptography libprotobuf ncurses pcre matplotlib numpy-base cffi harfbuzz pyqt pyparsing:  15%|▏| 13/87Examining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip ld_impl_linux-64 py-cpuinfo paddlepaddle-gpu pillow mkl_random pip urllib3 idna nltk wheel six cryptography matplotlib numpy-base cffi pyqt pyparsing:  15%|▏| 13/87 [00:46<04:37,  3.75s/it]                                                                                                                                              Examining conflict for pycparser pyyaml olefile python kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests matplotlib-base certifi sip ld_impl_linux-64 py-cpuinfo paddlepaddle-gpu pillExamining conflict for pycparser pyyaml olefile kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests sip certifi py-cpuinfo paddlepaddle-gpu pillow mkl_random pip urllib3 idna nltk wheeExamining conflict for pycparser pyyaml olefile kiwisolver numpy decorator python-dateutil mkl-service tornado mkl_fft scipy pysocks pyopenssl asn1crypto setuptools cycler protobuf chardet requests sip certifi py-cpuinfo paddlepaddle-gpu pillow mkl_random pip urllib3 idna nltk wheeExamining conflict for gstreamer pyqt gst-plugins-base qt:  17%|███████▊                                     | 15/87 [00:54<04:33,  3.79s/it]                                                                                                                                             Examining conflict for pyyaml gstreamer jpeg python libxcb kiwisolver graphviz libgcc-ng libxml2 numpy icu zlib mkl-service tornado mkl_fft scipy pixman libpng libedit zstd cairo fontconfig fribidi protobuf pango matplotlib-base freetype sip yaml expat openssl libtiff readline graphite2 sqlite pillow xz mkl_random dbus gst-plugins-base qt tk glib libuuid _libgcc_mutex libffi cryptography libprotobuf ncurses pcre matplotExamining conflict for pyyaml gstreamer jpeg python libxcb kiwisolver graphviz libgcc-ng libxml2 numpy icu zlib mkl-service tornado mkl_fft scipy pixman libpng libedit zstd cairo fontconfig fribidi protobuf pango matplotlib-base freetype sip yaml expat openssl libtiff readline graphite2 sqlite pillow xz mkl_random dbus gst-plugins-base qt tk glib libuuid _libgcc_mutex libffi cryptography libprotobuf ncurses pcre matplotExamining conflict for paddlepaddle-gpu gstreamer cairo graphviz dbus pango gst-plugins-base pcre qt harfbuzz glib:  20%|▏| 17/87 [00:56<03:07,  2.68s/it]                                                                                                                                                                                                                                                                             Examining conflict for paddlepaddle-gpu gstreamer cairo graphviz dbus pango gst-plugins-base pcre qt harfbuzz glib:  21%|▏| 18/87 [00:56<03:0Examining conflict for paddlepaddle-gpu gstreamer cairo graphviz dbus pango gst-plugins-base qt harfbuzz pyqt glib:  21%|▏| 18/87 [00:57<03:0Examining conflict for paddlepaddle-gpu gstreamer cairo graphviz dbus pango gst-plugins-base qt harfbuzz pyqt glib:  22%|▏| 19/87 [00:57<02:1Examining conflict for libtiff paddlepaddle-gpu jpeg pillow graphviz qt pyqt:  22%|█████▋                    | 19/87 [00:57<02:16,  2.00s/it]Examining conflict for wheel pyopenssl paddlepaddle-gpu setuptools python pip protobuf numpy matplotlib-base matplotlib urllib3 cryptography Examining conflict for wheel pyopenssl paddlepaddle-gpu setuptools python pip protobuf numpy matplotlib-base matplotlib urllib3 cryptography Examining conflict for libedit qt python sqlite:  24%|█████████████▎                                         | 21/87 [00:59<01:21,  1.24s/it]Examining conflict for paddlepaddle-gpu pillow cairo fontconfig graphviz pango matplotlib-base freetype qt matplotlib harfbuzz pyqt nltk:  32Examining conflict for paddlepaddle-gpu pillow cairo fontconfig graphviz pango matplotlib-base freetype qt matplotlib harfbuzz pyqt nltk:  33Examining conflict for libpng paddlepaddle-gpu pillow cairo fontconfig graphviz pango matplotlib-base freetype qt matplotlib harfbuzz pyqt nlExamining conflict for libpng paddlepaddle-gpu pillow cairo fontconfig graphviz pango matplotlib-base freetype qt matplotlib harfbuzz pyqt nlExamining conflict for paddlepaddle-gpu cairo fontconfig graphviz pango qt harfbuzz pyqt:  34%|████▊         | 30/87 [01:03<00:45,  1.25it/s]Examining conflict for paddlepaddle-gpu cairo fontconfig graphviz libxml2 pango matplotlib-base icu qt matplotlib harfbuzz pyqt nltk:  41%|▍|Examining conflict for graphite2 paddlepaddle-gpu graphviz pango qt harfbuzz:  43%|███████████               | 37/87 [01:07<00:27,  1.79it/s]Examining conflict for pyopenssl tornado paddlepaddle-gpu six mkl_random cycler numpy protobuf python-dateutil matplotlib-base matplotlib numExamining conflict for pyopenssl tornado paddlepaddle-gpu six mkl_random cycler numpy protobuf python-dateutil matplotlib-base matplotlib numExamining conflict for paddlepaddle-gpu mkl_random numpy matplotlib matplotlib-base numpy-base mkl-service mkl_fft nltk scipy:  56%|▌| 49/87 Examining conflict for paddlepaddle-gpu mkl_random numpy matplotlib matplotlib-base numpy-base mkl-service mkl_fft nltk scipy:  57%|▌| 50/87 Examining conflict for paddlepaddle-gpu mkl_random numpy matplotlib matplotlib-base mkl-service mkl_fft nltk scipy:  57%|▌| 50/87 [01:15<00:3Examining conflict for paddlepaddle-gpu mkl_random numpy matplotlib matplotlib-base mkl-service mkl_fft nltk scipy:  59%|▌| 51/87 [01:15<00:4Examining conflict for mkl_random intel-openmp numpy numpy-base mkl-service mkl_fft scipy mkl:  59%|█████▎   | 51/87 [01:17<00:47,  1.31s/it]Examining conflict for paddlepaddle-gpu mkl_random numpy matplotlib matplotlib-base numpy-base mkl-service mkl_fft nltk scipy mkl:  60%|▌| 52Examining conflict for paddlepaddle-gpu mkl_random numpy matplotlib matplotlib-base numpy-base mkl-service mkl_fft nltk scipy mkl:  61%|▌| 53Examining conflict for paddlepaddle-gpu blas mkl_random numpy matplotlib matplotlib-base numpy-base mkl-service mkl_fft nltk scipy:  61%|▌| 5Examining conflict for paddlepaddle-gpu blas mkl_random numpy matplotlib matplotlib-base numpy-base mkl-service mkl_fft nltk scipy:  62%|▌| 5Examining conflict for paddlepaddle-gpu mkl_random numpy matplotlib matplotlib-base numpy-base mkl-service mkl_fft nltk scipy libgfortran-ng:Examining conflict for paddlepaddle-gpu mkl_random numpy matplotlib matplotlib-base numpy-base mkl-service mkl_fft nltk scipy libgfortran-ng:Examining conflict for paddlepaddle-gpu mkl_random numpy matplotlib mkl-service mkl_fft nltk scipy:  63%|██▌ | 55/87 [01:27<01:11,  2.24s/it]Examining conflict for wheel tornado paddlepaddle-gpu setuptools pip protobuf requests matplotlib-base matplotlib certifi urllib3 cryptographExamining conflict for wheel tornado paddlepaddle-gpu setuptools pip protobuf requests matplotlib-base matplotlib certifi urllib3 cryptographExamining conflict for paddlepaddle-gpu cycler matplotlib matplotlib-base nltk:  77%|██████████████████▍     | 67/87 [01:36<00:09,  2.05it/s]failed                                                                                                                                    /  

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/cli/install.py", line 265, in install
        should_retry_solve=(_should_retry_unfrozen or repodata_fn != repodata_fns[-1]),
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 117, in solve_for_transaction
        should_retry_solve)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 158, in solve_for_diff
        force_remove, should_retry_solve)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 281, in solve_final_state
        ssc = self._run_sat(ssc)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
        return f(*args, **kwds)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 808, in _run_sat
        should_retry_solve=ssc.should_retry_solve
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
        return f(*args, **kwds)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 1316, in solve
        raise UnsatisfiableError({})
    conda.exceptions.UnsatisfiableError: 
    Did not find conflicting dependencies. If you would like to know which
    packages conflict ensure that you have enabled unsatisfiable hints.
    
    conda config --set unsatisfiable_hints True
                
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/exceptions.py", line 1079, in __call__
        return func(*args, **kwargs)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/cli/main.py", line 84, in _main
        exit_code = do_call(args, p)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/cli/conda_argparse.py", line 82, in do_call
        return getattr(module, func_name)(args, parser)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/cli/main_install.py", line 20, in execute
        install(args, parser, 'install')
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/cli/install.py", line 299, in install
        should_retry_solve=(repodata_fn != repodata_fns[-1]),
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 117, in solve_for_transaction
        should_retry_solve)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 158, in solve_for_diff
        force_remove, should_retry_solve)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 281, in solve_final_state
        ssc = self._run_sat(ssc)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
        return f(*args, **kwds)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 808, in _run_sat
        should_retry_solve=ssc.should_retry_solve
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
        return f(*args, **kwds)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 1318, in solve
        self.find_conflicts(specs, specs_to_add, history_specs)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 347, in find_conflicts
        bad_deps = self.build_conflict_map(specs, specs_to_add, history_specs)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 507, in build_conflict_map
        root, search_node, dep_graph, num_occurances)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 369, in breadth_first_search_for_dep_graph
        last_spec = MatchSpec.union((path[-1], target_paths[-1][-1]))[0]
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 481, in union
        return cls.merge(match_specs, union=True)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 475, in merge
        reduce(lambda x, y: x._merge(y, union), group) if len(group) > 1 else group[0]
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 475, in <lambda>
        reduce(lambda x, y: x._merge(y, union), group) if len(group) > 1 else group[0]
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 502, in _merge
        final = this_component.union(that_component)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 764, in union
        return '|'.join(options)
    TypeError: sequence item 0: expected str instance, Channel found

`$ /home/junc-lin/anaconda3/bin/conda install cudnn=7.6.0 cudatoolkit=10.0`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_DEFAULT_ENV=pa
                CONDA_EXE=/home/junc-lin/anaconda3/bin/conda
             CONDA_PREFIX=/home/junc-lin/anaconda3/envs/pa
           CONDA_PREFIX_1=/home/junc-lin/anaconda3
           CONDA_PREFIX_2=/home/junc-lin/anaconda3/envs/p_test
           CONDA_PREFIX_3=/home/junc-lin/anaconda3/envs/paddle
           CONDA_PREFIX_4=/home/junc-lin/anaconda3/envs/ppp
    CONDA_PROMPT_MODIFIER=(pa)
         CONDA_PYTHON_EXE=/home/junc-lin/anaconda3/bin/python
               CONDA_ROOT=/home/junc-lin/anaconda3
              CONDA_SHLVL=5
            DEFAULTS_PATH=/usr/share/gconf/ubuntu.default.path
           MANDATORY_PATH=/usr/share/gconf/ubuntu.mandatory.path
                     PATH=/home/junc-lin/anaconda3/bin:/home/junc-
                          lin/anaconda3/envs/pa/bin:/home/junc-
                          lin/anaconda3/condabin:/home/junc-lin/bin:/home/junc-lin/.local/bin:/u
                          sr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:
                          /usr/local/games:/snap/bin:/home/junc-lin/anaconda3/bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>
            XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0
         XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session2

     active environment : pa
    active env location : /home/junc-lin/anaconda3/envs/pa
            shell level : 5
       user config file : /home/junc-lin/.condarc
 populated config files : /home/junc-lin/.condarc
          conda version : 4.8.2
    conda-build version : 3.18.9
         python version : 3.7.4.final.0
       virtual packages : __cuda=10.1
                          __glibc=2.23
       base environment : /home/junc-lin/anaconda3  (writable)
           channel URLs : https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/Paddle/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/Paddle/noarch
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/noarch
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/junc-lin/anaconda3/pkgs
                          /home/junc-lin/.conda/pkgs
       envs directories : /home/junc-lin/anaconda3/envs
                          /home/junc-lin/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.2 requests/2.22.0 CPython/3.7.4 Linux/4.15.0-66-generic ubuntu/16.04.6 glibc/2.23
                UID:GID : 1001:1001
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.

If submitted, this report will be used by core maintainers to improve
future releases of conda.
Would you like conda to send this report to the core maintainers?

[y/N]: y
Upload successful.

Thank you for helping to improve conda.
Opt-in to always sending reports (and not see this message again)
by running

    $ conda config --set report_errors true

**Comment by chaogehah** [comment]
> > > in [pytorch/pytorch#17543](https://github.com/pytorch/pytorch/issues/17543) ：
> > > CUDA 9 and RTX 2080 Ti simply aren't compatible and dont play well togethere.
> > > An older CuDNN version working is likely a side-effect rather than expectation.
> > > Use CUDA10 and CUDA10 versions of CuDNN etc. for RTX 2080 which is Turing architecture.
> > > Can you try upgrading CUDA to 10.0?
> > > Can you use docker? Can you try the following docker environment？
> > > 
> > > * docker pull paddlepaddle/paddle:latest-gpu-cuda10.0-cudnn7-dev
> > 
> > 
> > 1.我尝试了升级到CUDA10.0，并且安装了paddle的cuda10.0的版本，但是当我输入完conda install paddlepaddle-gpu cudatoolkit=10.0并且安装完成后，其安装的cudnn版本为7.3.1无法使用，并且我在当前环境下无法安装别的版本的cudnn，一安装就出错。然后我尝试先安装cudnn7.6.0，但是装完之后paddle就无法安装了，这是我目前很头疼的一个问题，就是paddle跟cudnn的无法同时安装正确的版本，请问有别的什么好的办法吗？
> > 2.docker的话我自己没有用过，然后我对conda用的稍微熟练一点，如果能在conda下解决这个问题的话是最好的，若是实在没办法我再想想能不能用docker安装一下
> 
> 安装paddle cuda 10.0版本，安装的cudnn版本为7.3.1无法使用的表现是什么？
> 先安装cudnn7.6.0，为什么paddle无法安装？有什么报错么？

先安装cudnn时，报错是这样的

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: \ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
Examining conflict for sqlite paddlepaddle-gpu pip ncurses wheel certifi libedit python readline setuptools:  12%|▏| 3/24 [00:00<00:06,  3.44Examining conflict for openssl libgcc-ng sqlite paddlepaddle-gpu xz zlib pip ncurses wheel tk certifi libedit python readline setuptools libfExamining conflict for openssl libgcc-ng sqlite paddlepaddle-gpu xz zlib pip ncurses wheel tk certifi libedit python readline setuptools libfExamining conflict for openssl libgcc-ng sqlite _libgcc_mutex xz zlib ncurses libedit tk python readline libffi:  21%|▏| 5/24 [00:01<00:05,  Examining conflict for openssl libgcc-ng sqlite _libgcc_mutex xz zlib ncurses libedit tk python readline libffi:  25%|▎| 6/24 [00:01<00:05,  Examining conflict for paddlepaddle-gpu xz pip certifi wheel python setuptools:  25%|██████▎                  | 6/24 [00:01<00:05,  3.08it/s]failed                                                                                                                                       

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/cli/install.py", line 265, in install
        should_retry_solve=(_should_retry_unfrozen or repodata_fn != repodata_fns[-1]),
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 117, in solve_for_transaction
        should_retry_solve)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 158, in solve_for_diff
        force_remove, should_retry_solve)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 281, in solve_final_state
        ssc = self._run_sat(ssc)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
        return f(*args, **kwds)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 808, in _run_sat
        should_retry_solve=ssc.should_retry_solve
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
        return f(*args, **kwds)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 1316, in solve
        raise UnsatisfiableError({})
    conda.exceptions.UnsatisfiableError: 
    Did not find conflicting dependencies. If you would like to know which
    packages conflict ensure that you have enabled unsatisfiable hints.
    
    conda config --set unsatisfiable_hints True
                
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/exceptions.py", line 1079, in __call__
        return func(*args, **kwargs)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/cli/main.py", line 84, in _main
        exit_code = do_call(args, p)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/cli/conda_argparse.py", line 82, in do_call
        return getattr(module, func_name)(args, parser)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/cli/main_install.py", line 20, in execute
        install(args, parser, 'install')
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/cli/install.py", line 299, in install
        should_retry_solve=(repodata_fn != repodata_fns[-1]),
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 117, in solve_for_transaction
        should_retry_solve)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 158, in solve_for_diff
        force_remove, should_retry_solve)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 281, in solve_final_state
        ssc = self._run_sat(ssc)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
        return f(*args, **kwds)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 808, in _run_sat
        should_retry_solve=ssc.should_retry_solve
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
        return f(*args, **kwds)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 1318, in solve
        self.find_conflicts(specs, specs_to_add, history_specs)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 347, in find_conflicts
        bad_deps = self.build_conflict_map(specs, specs_to_add, history_specs)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 507, in build_conflict_map
        root, search_node, dep_graph, num_occurances)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 369, in breadth_first_search_for_dep_graph
        last_spec = MatchSpec.union((path[-1], target_paths[-1][-1]))[0]
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 481, in union
        return cls.merge(match_specs, union=True)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 475, in merge
        reduce(lambda x, y: x._merge(y, union), group) if len(group) > 1 else group[0]
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 475, in <lambda>
        reduce(lambda x, y: x._merge(y, union), group) if len(group) > 1 else group[0]
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 502, in _merge
        final = this_component.union(that_component)
      File "/home/junc-lin/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 764, in union
        return '|'.join(options)
    TypeError: sequence item 0: expected str instance, Channel found

`$ /home/junc-lin/anaconda3/bin/conda install paddlepaddle-gpu cudatoolkit=10.0`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_DEFAULT_ENV=ppp
                CONDA_EXE=/home/junc-lin/anaconda3/bin/conda
             CONDA_PREFIX=/home/junc-lin/anaconda3/envs/ppp
           CONDA_PREFIX_1=/home/junc-lin/anaconda3
           CONDA_PREFIX_2=/home/junc-lin/anaconda3/envs/p_test
           CONDA_PREFIX_3=/home/junc-lin/anaconda3/envs/paddle
    CONDA_PROMPT_MODIFIER=(ppp)
         CONDA_PYTHON_EXE=/home/junc-lin/anaconda3/bin/python
               CONDA_ROOT=/home/junc-lin/anaconda3
              CONDA_SHLVL=4
            DEFAULTS_PATH=/usr/share/gconf/ubuntu.default.path
           MANDATORY_PATH=/usr/share/gconf/ubuntu.mandatory.path
                     PATH=/home/junc-lin/anaconda3/bin:/home/junc-
                          lin/anaconda3/envs/ppp/bin:/home/junc-
                          lin/anaconda3/condabin:/home/junc-lin/bin:/home/junc-lin/.local/bin:/u
                          sr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:
                          /usr/local/games:/snap/bin:/home/junc-lin/anaconda3/bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>
            XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0
         XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session2

     active environment : ppp
    active env location : /home/junc-lin/anaconda3/envs/ppp
            shell level : 4
       user config file : /home/junc-lin/.condarc
 populated config files : /home/junc-lin/.condarc
          conda version : 4.8.2
    conda-build version : 3.18.9
         python version : 3.7.4.final.0
       virtual packages : __cuda=10.1
                          __glibc=2.23
       base environment : /home/junc-lin/anaconda3  (writable)
           channel URLs : https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/Paddle/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/Paddle/noarch
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/noarch
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/junc-lin/anaconda3/pkgs
                          /home/junc-lin/.conda/pkgs
       envs directories : /home/junc-lin/anaconda3/envs
                          /home/junc-lin/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.2 requests/2.22.0 CPython/3.7.4 Linux/4.15.0-66-generic ubuntu/16.04.6 glibc/2.23
                UID:GID : 1001:1001
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.

If submitted, this report will be used by core maintainers to improve
future releases of conda.
Would you like conda to send this report to the core maintainers?

[y/N]: y
Upload successful.

Thank you for helping to improve conda.
Opt-in to always sending reports (and not see this message again)
by running

    $ conda config --set report_errors true

**Comment by JepsonWong** [comment]
我觉得你还是先用docker测试一下你的程序环境、显卡有没有问题。

**Comment by chaogehah** [comment]
> 我觉得你还是先用docker测试一下你的程序环境、显卡有没有问题。

好吧，那我回头学习一下docker看看，谢谢

**Comment by tianshuo78520a** [comment]
使用新版本的Paddle还会有问题么？比如1.8.2
